package.json

```diff
@@ -69,7 +69,7 @@
     "@griffel/webpack-loader": "2.2.10",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33235&commitId=4fef1b82ed235a5f059c91600edb62672ebd7b3b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Root.chromium.png|2309| Changed|
|Callout.Right center.chromium.png|2396| Changed|
|Callout.No beak.chromium.png|2306| Changed|
|Callout.Rendering callout attached to a rectangle.chromium.png|1493| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>